ownership-chain/pallets/collator-rewards/src/tests.rs

```diff
@@ -0,0 +1,59 @@
+use frame_support::traits::Currency;
+
+use crate::mock::*;
+
+#[test]
+fn test_block_rewards_work() {
+	new_test_ext().execute_with(|| {
+		let reward_per_block = RewardPerBlock::get();
+		let community_incentives_account = CommunityIncentivesAccountId::get();
+
+		// community incentives account has `INITIAL_BALANCE` balance
+		assert_eq!(Balances::free_balance(community_incentives_account), INITIAL_BALANCE);
+
+		// 4 is the default author
+		assert_eq!(Balances::free_balance(4), INITIAL_BALANCE);
+
+		// trigger `note_author` for block 1
+		initialize_to_block(1);
+
+		// 4 is the default author, so he is rewarded
+		assert_eq!(Balances::free_balance(4), INITIAL_BALANCE + reward_per_block);
+
+		// check for event
+		System::assert_has_event(RuntimeEvent::CollatorRewards(crate::Event::CollatorRewarded {
+			collator: 4,
+			amount: reward_per_block,
+		}));
+
+		// community incentives account's balance decreased
+		assert_eq!(
+			Balances::free_balance(community_incentives_account),
+			INITIAL_BALANCE - reward_per_block
+		);
+
+		// trigger `note_author` for block 2
+		initialize_to_block(2);
+
+		// 4 is the default author, so he is rewarded
+		assert_eq!(Balances::free_balance(4), INITIAL_BALANCE + reward_per_block * 2);
+
+		// leave less than `reward_per_block` in the community incentives account
```

Comment:
I suggest create a separated test to check when CI has less balance that reward amount